packages/datagateway-dataview/src/page/pageContainer.component.tsx

```diff
@@ -475,51 +485,64 @@ class PageContainer extends React.Component<
   };
 
   public render(): React.ReactElement {
-    return (
-      <Paper square elevation={0} style={{ backgroundColor: 'inherit' }}>
-        <NavBar
-          entityCount={this.props.entityCount}
-          cartItems={this.props.cartItems}
-          navigateToSearch={this.props.navigateToSearch}
-          navigateToDownload={this.props.navigateToDownload}
+    if (this.props.location.pathname === paths.homepage) {
+      return (
+        <HomePage
+          {...this.props}
+          logo={this.props.pluginHostUrl + DGLogo}
+          backgroundImage={this.props.pluginHostUrl + BackgroundImage}
+          exploreImage={this.props.pluginHostUrl + ExploreImage}
+          discoverImage={this.props.pluginHostUrl + DiscoverImage}
+          downloadImage={this.props.pluginHostUrl + DownloadImage}
```

Comment:
Can we do this using a `Switch` and a `Route` rather than manually via the redux store? I know we connect our router to the store but I think we should still do simple routing like this the `react-router` way. You'd then probably need to refactor the `else` into a standalone component to provide to a `Route` with no `path` param (basically catches all other routes). We need to refactor into a standalone component and not an inline render component because otherwise we'd have a lot of unnecessary rerenders.